rust/main/agents/scraper/src/db/generated/gas_payment.rs

```diff
@@ -21,6 +21,10 @@ pub struct Model {
     pub gas_amount: BigDecimal,
     pub tx_id: i64,
     pub log_index: i64,
+    pub origin: Option<i32>,
```

Comment:
optional because pre-existing entries don't have this column?

Reply:
Yes. Once we start to populate this field from code and do a backfill, there won't be null values and optional part can be removed.